src/js/core/Display.js

```diff
@@ -0,0 +1,36 @@
+export default class Display {
+  container;
+
+  constructor() {
+    if (this.constructor === Display) {
+      throw new Error('추상 클래스는 인스턴스화 할 수 없습니다.');
+    }
+    this.init();
+  }
+
+  init() {
+    this.setContainer();
+    this.defaultElement();
+    this.bindEvents();
+    this.subscribeStores();
+  }
```

Comment:
별도로 init 을 묶어줄 필요는 없어보입니다.

```suggestion
    this.setContainer();
    this.defaultElement();
    this.bindEvents();
    this.subscribeStores();
  }
```

Reply:
저는 나중에 코드 읽을 때 '아 이게 초기화 함수구나~'라는 걸 개발자가 쉽게 알게해주도록 init이란 메서드로 묶어줬었는데요, 생각해보니 생성자 자체가 초기화의 역할을 하기 때문에 굳이 분리해줄 필요가 없다는 것을 깨달았습니다.. :) 감사합니다!

[ad1659f](https://github.com/woowacourse/javascript-youtube-classroom/pull/96/commits/ad1659f18551a00f6ce4dac06c49f54451979fc9)